src/NServiceBus.Core/Transports/Msmq/QueuePermissions.cs

```diff
@@ -59,8 +59,6 @@ public static void SetPermissionsForQueue(MessageQueue queue, string account)
         {
 
             queue.SetPermissions(LocalAdministratorsGroupName, MessageQueueAccessRights.FullControl, AccessControlEntryType.Allow);
-            queue.SetPermissions(LocalEveryoneGroupName, MessageQueueAccessRights.WriteMessage, AccessControlEntryType.Allow);
```

Comment:
Ah, so we never needed Everyone/Anonymous in the first place? (wonder why we added it)